src/components/Collaborative/utils/createWebsocketConnection.ts

```diff
@@ -0,0 +1,7 @@
+import { WebsocketProvider } from "y-websocket";
+import * as Y from "yjs";
+
+export function createWebsocketConnection(hash: string) {
+    const ydoc = new Y.Doc();
+    return new WebsocketProvider("ws://localhost:1234", hash, ydoc);
```

Comment:
todo: create a task to transform this in a variable that we can pull from the config.yaml file